danger-kotlin/src/runnerMain/kotlin/com/danger/runner/cmd/danger-js/DangerJS.kt

```diff
@@ -0,0 +1,17 @@
+package com.danger.runner.cmd.`danger-js`
+
+import com.danger.runner.cmd.Cmd
+import com.danger.runner.cmd.ICmd
+
+object DangerJS: DangerJSBridge {
+    override val cmd: ICmd
+        get() = Cmd
+
+    override fun process(command: String, processName: String, args: List<String>) {
+        with(cmd) {
+            name("$(which danger-$command)")
```

Comment:
This is now changed, should be `which danger` otherwise will not work with brew (I know is not related to your change, but given is needed...)